Packs/FireEyeHX/Integrations/FireEyeHXv2/FireEyeHXv2.py

```diff
@@ -691,14 +691,35 @@ def get_token_request(self):
                 resp_type='response'
             )
         except Exception as e:
-            demisto.debug(f'Encountered an error for url {self._base_url}/token: {e}')
+            exception_str = str(e)
+            demisto.debug(f'Encountered an error for url {self._base_url}/token: {exception_str}')
+            if 'Incorrect user id or password' in exception_str:
+                raise DemistoException('Unauthorized - Incorrect user id or password')
             raise ValueError("Server URL incorrect")
```

Comment:
Is it just URL correctness here? It could be some networking problem, etc.
```suggestion
            raise ValueError("Could not get a token")
```